sale_order_invoice_amount/views/sale_order_view.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_order_form_invoiced_amount" model="ir.ui.view">
+        <field name="name">sale.order.form.invoiced.amount</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="arch" type="xml">
+            <field name="amount_untaxed" position="after">
+                <field name="invoiced_amount" />
+                <field name="uninvoiced_amount" />
+            </field>
+        </field>
+    </record>
+    <record id="view_quotation_tree_invoiced_amount" model="ir.ui.view">
+        <field name="name">sale.order.tree.invoiced_amount</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_quotation_tree" />
```

Comment:
You should inherit the SO list view (`view_order_tree`), I don't think it is needed in quoatations.